console/types/address/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub use snarkvm_console_types_boolean::Boolean;
 pub use snarkvm_console_types_field::Field;
 pub use snarkvm_console_types_group::Group;
 
+/// The unique ID of an Aleo account, which is derived from its view (public) key.
```

Comment:
View keys are not intended to be publicly shared

Reply:
I added this when trying to understand how to derive the ID from a private key.

Do you have suggestions on how to rephrase it? I can also remove it, as it is not closely related to the PR.

Reply:
I will just remove the "(public)" part for now.